src/OpenTok/Render.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace OpenTok;
+
+class Render
```

Comment:
Please add a description of the class (Represents an Experience Composer renderer).